README.md

```diff
@@ -3,6 +3,21 @@ A Python library for interfacing with the "ONZO" energy monitor
 
 Tested on Linux & MacOS
 
+# Installation
+This software requires that you use [Python 3.x](https://www.python.org/downloads/).
+
+To use this package you must also ensure you install hidapi.
+```
+sudo pip install hidapi
+```
+
+# Energy readings
+To get a basic understanding of the meter readings from your Onzo energy meter you can use ``device.py`` script to view data being broadcast over 433Mhz from your energy clamp and then read the data over usb connected to your computer.
```

Comment:
`read.py` does the readings, `device.py` just contains the class for talking with the device.

Reply:
Yeah, my bad I don't know why I used that filename. Making the changes now.